src/infrastructure/transaction/CreateTransactionFromDTO.ts

```diff
@@ -118,17 +118,27 @@ export const CreateTransactionFromDTO = (transactionDTO): Transaction => {
  */
 const CreateStandaloneTransactionFromDTO = (transactionDTO, transactionInfo): Transaction => {
     if (transactionDTO.type === TransactionType.TRANSFER) {
+        /**
+         * Check if message is encoded (from DTO) or is raw (from JSON)
+         */
+        let message = EmptyMessage;
+        if (transactionDTO.message !== undefined && convert.isHexString(transactionDTO.message.payload)) {
+            message = PlainMessage.createFromDTO(transactionDTO.message.payload);
```

Comment:
Can you rename to `createFromPayload` ? it is clearer in usage

Reply:
👍

Reply:
Done!